.teamcity/test-buckets.json

```diff
@@ -1949,19 +1949,19 @@
 			{
 				"parallelizationMethod":{
 					"name":"TeamCityParallelTests",
-					"numberOfBatches":7
+					"numberOfBatches":9
 				},
 				"subprojects":[
-					"configuration-cache"
+					"dependency-management"
 				]
 			},
 			{
 				"parallelizationMethod":{
 					"name":"TeamCityParallelTests",
-					"numberOfBatches":6
+					"numberOfBatches":7
 				},
 				"subprojects":[
-					"dependency-management"
+					"configuration-cache"
```

Comment:
🤔 Can we fix that in the generation logic so that we'd also do a particular order, maybe based on the name of the first subproject in the bucket? So we don't need to rely on manual sorting further down the road?